.travis.yml

```diff
@@ -1,8 +1,8 @@
 # Copy these contents into the root directory of your Github project in a file
 # named .travis.yml
 
-# Use new container infrastructure to enable caching
-sudo: false
+# Use sudo-enabled environment to get 7.5GB of memory instead of 4.
+sudo: required
```

Comment:
Seems to have worked.

[Here](https://docs.travis-ci.com/user/ci-environment/)'s an overview of the various Travis CI environments.

Not sure why the OS X jobs didn't error out too.